packages/action-group/test/action-group.test.ts

```diff
@@ -352,20 +349,20 @@ describe('ActionGroup', () => {
                 },
             ],
         });
+        await opened;
 
-        await elementUpdated(el);
+        await aTimeout(100);
 
-        const opened = oneEvent(el.children[3] as ActionMenu, 'sp-opened');
-        await opened;
+        expect(el.children[3]).to.equal(document.activeElement);
+        //todo test times out waiting for menu close
+        //const closed = oneEvent(el.children[3] as ActionMenu, 'sp-closed');
 
         // use keyboard to navigate to the second menu item and select it
         await sendKeys({ press: 'ArrowDown' });
         await sendKeys({ press: 'Enter' });
 
-        const closed = oneEvent(el.children[3] as ActionMenu, 'sp-closed');
-        await closed;
+        //await closed
```

Comment:
not sure why `closed` is timing out here

Reply:
Is this something you are still figuring out or you want to tackle this in a separate PR?

Reply:
Solved it! :)